types/office-js-preview/index.d.ts

```diff
@@ -24368,5615 +24368,5615 @@ declare namespace Excel {
-        * @beta
-        */
+         * An error caused by the user exceeding their quota of concurrent connections to external code service environments. Displays as error type #BLOCKED! in Excel.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         environmentConnectionExceedQuota = "EnvironmentConnectionExceedQuota",
         /**
-        * An error caused by an environment definition not being supported by an external code service. Displays as error type #BLOCKED! in Excel.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * An error caused by an environment definition not being supported by an external code service. Displays as error type #BLOCKED! in Excel.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         unsupportedEnvironment = "UnsupportedEnvironment",
         /**
-        * An error caused by function being unavailable in Partial or Manual calculation modes. Displays as error type #BLOCKED! in Excel.
-        *
-        * @remarks
-        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
-        * @beta
-        */
+         * An error caused by function being unavailable in Partial or Manual calculation modes. Displays as error type #BLOCKED! in Excel.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
         unsupportedInCalculationMode = "UnsupportedInCalculationMode"
     }
     /**
-    * Represents the value of a cell containing a #BLOCKED! error.
-    *
-    * @remarks
-    * [Api set: ExcelApi 1.16]
-    */
+     * Represents the value of a cell containing a #BLOCKED! error.
+     *
+     * @remarks
+     * [Api set: ExcelApi 1.16]
+     */
     interface BlockedErrorCellValue {
         /**
-        * Represents the type of this cell value.
-        *
-        * @remarks
-        * [Api set: ExcelApi 1.16]
-        */
+         * Represents the type of this cell value.
+         *
+         * @remarks
+         * [Api set: ExcelApi 1.16]
+         */
         type: CellValueType.error | "Error";
         /**
-        * Represents the value that would be returned by `Range.values` for a cell with this value.
-        * When accessed through a `valuesAsJson` property, this string value aligns with the en-US locale.
-        * When accessed through a `valuesAsJsonLocal` property, this string value aligns with the user's display locale.
-        *
-        * @remarks
-        * [Api set: ExcelApi 1.16]
-        */
+         * Represents the value that would be returned by `Range.values` for a cell with this value.
+         * When accessed through a `valuesAsJson` property, this string value aligns with the en-US locale.
+         * When accessed through a `valuesAsJsonLocal` property, this string value aligns with the user's display locale.
```

Comment:
I'm not a custom functions expert, but will devs understand what it means when a property is "accessed through" another property? (I don't.)

Reply:
This phrasing has been released to GA for over 2 years and we haven't gotten any questions about it, so I think it's ok.